### DISCUSSION TITLE
electron-app arm64: dev works, build not

### DISCUSSION THREAD

**Discussion by kleineLoesungen** [quasarframework/quasar#13211] [CLI - Electron mode] [answered]
Hello,

my dev environment with `q-select`works fine, but after build it is not displayed. Same with an input (type: file) field. I have nothing changed at default build configuration from quasar.

I am a beginner and new at quasar/electron development.
Any ideas? If you need more informations, please describe where to find them.

Thank you
Sebastian

**Comment by metalsadman** [comment]
no error log in your dev console? try the build command with debug flag ie. `quasar build -m electron --debug` to see if it shows any error.

**Reply by kleineLoesungen** [reply]
unfortunately not.

**dev**
<img width="606" alt="Bildschirmfoto 2022-04-19 um 06 08 00" src="https://user-images.githubusercontent.com/63931750/163918645-3511771e-2549-4bae-9c6b-04ca3ba94844.png">

**build**
<img width="612" alt="Bildschirmfoto 2022-04-19 um 06 08 27" src="https://user-images.githubusercontent.com/63931750/163918662-c217b3cd-4d3e-427b-aeb3-9a45b09b335c.png">

**build --debug**
<img width="670" alt="Bildschirmfoto 2022-04-19 um 06 09 28" src="https://user-images.githubusercontent.com/63931750/163918681-70cc4879-8e57-4cd2-90c8-720246e7860e.png">

code part
```
<!-- filter -->
        <div class="col">
            <q-select
                filled
                v-model="selectModel"
                :options="selectElementOptions"
                color="teal"
                dense
                hide-selected
                emit-value
                use-input
                @filter="selectFilter"
                behavior="dialog"
            >
                <template v-slot:prepend>
                    <q-icon name="search" />
                </template>

                <template v-slot:option="scope">
                    <q-item v-bind="scope.itemProps">
                        <q-item-section>
                            <q-item-label>{{ scope.opt.label }}</q-item-label>
                            <q-item-label caption>
                                <q-badge class="q-mr-sm" color="red-4" v-for="tag in scope.opt.tags" :key="tag">
                                    {{ tag }}
                                </q-badge>
                            </q-item-label>
                        </q-item-section>
                        <q-item-section v-if="scope.opt.match" side>
                            <span class="text-weight-light text-right">found in</span>
                            <q-badge rounded color="red-2">{{ scope.opt.match }}</q-badge>
                        </q-item-section>
                    </q-item>
                </template>

                <template v-slot:no-option>
                    <q-item>
                        <q-item-section class="text-grey">
                        No results
                        </q-item-section>
                    </q-item>
                </template>
            </q-select>
        </div>
```

in dev-debug I only have a 3 typescript-warnings about any type. One of them is related to `@filter="selectFilter"`

**Reply by MilosPaunovic** [reply]
First thing is to amend those warnings and try again. If that does not help, please provide a minimal code reproduction repository for someone to be able to take a closer look, as this  further on would be tapping in the dark.

**Reply by kleineLoesungen** [reply]
Thanks for your quick feedback. I will try to fix them and get back with the result.

**Reply by kleineLoesungen** [reply]
Now I have access to `--debug` and see `Failed to load resource: net::ERR_CONNECTION_REFUSED (localhost/:1)`
but I found nothing helpful

**Reply by kleineLoesungen** [reply]
@MilosPaunovic @metalsadman have you any ideas? If I comment out everything, I got this error message too.

**Reply by MilosPaunovic** [reply]
Have you properly added Electron mode? Best way forward would be for you to provide minimal reproduction repository, so someone can take a closer look at it.

**Reply by kleineLoesungen** [reply]
> Now I have access to `--debug` and see `Failed to load resource: net::ERR_CONNECTION_REFUSED (localhost/:1)`
> but I found nothing helpful

I found in header `<script src="http://localhost:8098"></script>`. Don' ask me why I had add this line, but after removing the error disappeared.

**Comment by kleineLoesungen** [comment, marked as answer]
I created a new project and copied my app by steps. Now It works. Maybe some remains from tests.

Thanks for help!
